client/src/components/ReportCollection.js

```diff
@@ -38,6 +41,9 @@ const ReportCollection = ({
   const [numberOfPeriods, setNumberOfPeriods] = useState(3)
   const contRef = useResponsiveNumberOfPeriods(setNumberOfPeriods)
   const [viewFormat, setViewFormat] = useState(viewFormats[0])
+  const [calendarAttendeeType, setCalendarAttendeeType] = useState(
+    Settings.calendarOptions.attendeesType
```

Comment:
You're using this setting unconditionally, yet the schema does not require the `calendarOptions` to be present in the dictionary.